tests/e2e/scenarios/test_reborn_webui_v2_legacy_core.py

```
@@ -0,0 +1,286 @@
+              }, 0);
+            }
+            close() {
+              this.readyState = 2;
+            }
+          }
+          window.EventSource = FakeEventSource;
+        })();
+        """
+    )
+
+    async def fulfill_json(route, body, status=200):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `fulfill_json`

Add return type annotation: `None`

(ANN202)